open_spiel/python/egt/alpharank.py

```diff
@@ -791,3 +791,41 @@ def compute(payoff_tables,
     print_results(payoff_tables, payoffs_are_hpt_format, rhos, rho_m, c, pi)
 
   return rhos, rho_m, pi, num_profiles, num_strats_per_population
+
+def suggest_alpha(payoff_tables):
+  """Suggests an alpha for use in alpha-rank. 
+
+  The suggested alpha is approximately the smallest possible alpha such that 
+  the ranking has 'settled out'. It is calculated as 2/min_gap_between_payoffs. 
+
+  Args:
+    payoff_tables: List of game payoff tables, one for each agent identity. Each
+      payoff_table may be either a numpy array, or a _PayoffTableInterface
+      object.
+  
+  Returns:
+    A suggested alpha.
+  """ 
+  num_strats_per_population =\
+    utils.get_num_strats_per_population(payoff_tables, False)
+  num_profiles = utils.get_num_profiles(num_strats_per_population)
+
+  gap = np.inf
+  for id_row_profile in range(num_profiles):
+    row_profile = utils.get_strat_profile_from_id(num_strats_per_population,
+                                                id_row_profile)
+
+    next_profile_gen = utils.get_valid_next_profiles(num_strats_per_population,
+                                                    row_profile)
+
+    for index_population_that_changed, col_profile in next_profile_gen:
+      payoff_table_k = payoff_tables[index_population_that_changed]
+      f_r = _get_payoff(payoff_table_k, False, col_profile, 
+                          index_population_that_changed)
+      f_s = _get_payoff(payoff_table_k, False, row_profile, 
+                          index_population_that_changed)
+      if f_r > f_s:
+        gap = min(gap, f_r - f_s)
+
+  return 2/gap
```

Comment:
Since the '2' here controls the series decay rate tolerance, perhaps make it an arg (with a default value of 2, which seems sensible!)